extension/js/common/composer.ts

```diff
@@ -903,6 +903,7 @@ export class Composer {
     if (Catch.browser().name === 'firefox') { // the padding cause issues in firefox where user cannot click on the message password
       this.S.cached('input_text').css({ 'padding-top': 0, 'padding-bottom': 0 });
     }
+    this.S.cached('body').keydown(Ui.escape(() => !this.composeWindowIsMinimized && this.app.closeMsg()));
```

Comment:
My recommendation here is to manually trigger the event like so: `$('.close_new_message').click()` and let the handler worry about it, for two reasons:
1) there are additional checks in that handler, and this way all paths will go through the same "chokepoint". See the handler:
```js
      $('.close_new_message').click(Ui.event.handle(async () => {
        if (!this.isSendMessageInProgress || await Ui.modal.confirm('A message is currently being sent. Closing the compose window may abort sending the message.\nAbort sending?')) {
          this.app.closeMsg();
        }
      }, this.getErrHandlers(`close message`)));
```
2) It should not trigger for message replies. I believe, doing what I'm suggesting, it won't. I'm guessing, though have not tested it, that your method may (or it may close another open new message, although you are focused onto a reply frame? That could be good or bad, but either way somewhat unexpected).

Please try the above suggestion, and then test that it does not affect a reply window, and test that if you have a reply window focused, it should not (I think?) affect other open compose windows.

Reply:
> My recommendation here is to manually trigger the event like so: $('.close_new_message').click() and let the handler worry about it

Makes sense, done!

> It should not trigger for message replies

It was closing by Esc, so I added the additional check in https://github.com/FlowCrypt/flowcrypt-browser/pull/1922/commits/5346285d668d6b49d590cc9478b700dcd84b933a

>  test that if you have a reply window focused, it should not (I think?) affect other open compose windows.

Tested, works nicely :+1: